spec/javascripts/govuk_publishing_components/modules.spec.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-env jasmine, jquery */
+
+var $ = window.jQuery
+
+describe('GOVUK Modules', function () {
```

Comment:
Was there any documentation accompanying the modules code? If not, should we write some? Feels like a good opportunity (although probably in a separate PR).

Reply:
Good point. There's the [Modules section](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules) in the JavaScript doc. I'll extract it and add it to this repo.